addons/scratchblocks/userscript.js

```diff
@@ -30,6 +26,14 @@ async function getLocales(addon) {
     36: "id",
     59: "fa",
   };
+
+  if (location.pathname.split("/")[2] === "settings") {
+    return ["en", locale]
```

Comment:
Vanilla behavior is to use English only, we should do the same

Reply:
done